docs/README.md

```diff
@@ -9,55 +9,55 @@ using automatically managed compute and memory resources. Composer is
 
-Composer has two parts. The first is a library for describing
-compositions, programmatically. The library is currently available in
-Node.js. The second is a runtime that executes the composition. We
-will explain these components in greater detail, but first, we will
-introduce you to the programming environment for compositions.
-
 Programming for the serverless cloud is a uniquely new experience. For
 this reason, we have developed a unified environment that offers the
 benefits and familiarity of a command line interface, with
 visualization and a graphical interface to assist in certain
-tasks. This environment is offered through a new tool called `fsh`:
-_the functions programming shell for the IBM Cloud_.
-
-- [Programming shell quick start](#programming-shell-quick-start)
-  - [Installing the shell](#installing-the-shell)
-  - [Tour of the command line tool](#tour-of-the-programming-shell)
-  - [Getting setup to run compositions](#before-you-run-your-first-app)
-
-- Your first composition
-  - [Create a composition](#your-first-composition)
-  - [Preview your composition](#previewing-a-composition)
-  - [Deploy and run](#running-your-first-app)
-  - [Visualize an execution](#visualizing-sessions)
-  - [Composing OpenWhisk actions](#composing-openwhisk-actions)
-
-- [Compositions by example](#compositions-by-example)
-  - [if-then-else](#if-combinator)
-  - [try-catch](#try-combinator)
-  - [data forwarding](#nesting-and-forwarding)
-  - [scoped variables](#variables-and-scoping)
-
-- The Composer programming model
-  - [The composition library](COMPOSER.md)
-  - [The underlying composition model](FORMAT.md)
-  - [The execution model](CONDUCTOR.md)
-
-## Programming shell quick start
-
-The programming shell for functions and compositions is a new
-developer experience with fluid migration between a conventional
-command line tool and a graphical interface. It is also the
-environment for developing and working with serverless compositions.
-
-### Installing the shell
-
-The programming shell is currently distributed through the [Node
+tasks. This environment is offered through a new tool called 
+[IBM Cloud Shell](https://github.com/ibm-functions/shell), or just _Shell_.
+
+## Shell quick start
+
+Shell is a cross-platform desktop application powered by [Electron](https://electronjs.org/). 
+
+### Before you run your first app
+
+You must have a valid IBM Cloud (i.e., Bluemix)
+[account](https://ibm.biz/openwhisk), or deploy [Apache
+OpenWhisk](https://github.com/apache/incubator-openwhisk)
+locally. This is needed because Composer builds on and extends Apache
+OpenWhisk, which powers IBM Cloud Functions.
+
+* _Using composer with IBM Cloud Functions:_ you need an IBM Cloud
+[account](https://ibm.biz/openwhisk), and the [IBM Cloud CLI](https://console.bluemix.net/docs/cli/reference/bluemix_cli/download_cli.html#download_install) (`bx`). You will also need to install the Cloud Function Plugin for bx: 
```

Comment:
You don’t actually need to install and configure bx... I think this is an unnecessary overhead. 

If you already are a Functions or openwhisk user there’s nothing to do. If you’re going to go and install bx you can just as easily get your key from the Functions ui and plug it in.  

It might be nice to get that as a option... But OK...

Reply:
The UI no longer offers keys. They direct you to be I think?

Reply:
It might be hidde under how do I make an http request. There was a feature request to restore it also. But if you’re right, you’re right.

Reply:
here you go: https://console.bluemix.net/openwhisk/learn/api-key

Reply:
Hehe. It indeed might be hidden somewhere. I'll look again more closely. It certainly wasn't at the top level last I checked.

Reply:
I wonder whether we could screen scrape this page on the side car.

Reply:
I'll add a note to say if you are already a `wsk` or `bx wsk` user you don't need to do anything. You only need to do those steps if you are new to cloud functions/openwhisk. These instructions are copied from https://console.bluemix.net/openwhisk/learn/cli